### PR TITLE
fix incorrect mp-jwt message

### PR DESCRIPTION
**Pull request by brutif**

### CODE REVIEW

**Comment by brutif** [comment]
It was noted that the mp-jwt attribute name for the signer certificate alias is inconsistent with the jwt consumer name.  The message needs to be fixed to be useful in both scenarios.

**Comment by brutif** [comment]
#build 
#spawn.fullfat.buckets=com.ibm.ws.security.jwt_fat.builder,com.ibm.ws.security.jwt_fat.consumer,com.ibm.ws.security.jwt_fat.mpjwt

File: dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops [hunk 1]
```diff
@@ -277,9 +277,9 @@ JWT_SIGNER_CERT_NOT_AVAILABLE.useraction=Verify that the truststore that is used
...
-JWT_SIGNER_CERT_AMBIGUOUS.explanation=Multiple signer certificates were found in the configured truststore. When there are multiple certificates in the truststore, the 'trustedAlias' attribute must be specified in the JWT consumer configuration to establish which of the certificates to use.
-JWT_SIGNER_CERT_AMBIGUOUS.useraction=Specify the 'trustedAlias' JWT consumer configuration attribute with the alias of the appropriate signer certificate.
+JWT_SIGNER_CERT_AMBIGUOUS=CWWKS6047E: Cannot determine which signer certificate in the truststore to use. Add the 'trustedAlias' attribute to the JWT consumer configuration or the 'keyName' attribute to the MP-JWT configuration.
+JWT_SIGNER_CERT_AMBIGUOUS.explanation=Multiple signer certificates were found in the configured truststore. When there are multiple certificates in the truststore, the 'trustedAlias' attribute must be specified in the JWT consumer configuration or the 'keyName' attribute must be specified in the MP-JWT configuration to establish which of the certificates to use.
```
Comment (lzdanski) [inline dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops]: +JWT_SIGNER_CERT_AMBIGUOUS.explanation=Multiple signer certificates were found in the configured truststore. When there are multiple certificates in the truststore, the 'trustedAlias' attribute must be specified in the JWT consumer configuration or the 'keyName' attribute must be specified in the MP-JWT configuration to establish which **certificate to use.**

File: dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops [hunk 2]
```diff
@@ -277,9 +277,9 @@ JWT_SIGNER_CERT_NOT_AVAILABLE.useraction=Verify that the truststore that is used
...
+JWT_SIGNER_CERT_AMBIGUOUS.useraction=Specify the 'trustedAlias' JWT consumer configuration attribute or the 'keyName' MP-JWT configuration attribute with the alias of the appropriate signer certificate.
```
Comment (lzdanski) [inline dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops]: I was a little confused here but I think I understand. To fix the ambiguity, the user needs to specify which certificate in the truststore should be used. Will adding the attribute to either 'kevNAME' or 'trustedAlias' produce the same effect? I've rephrased the message a little to make the relationship a little clearer that you can fix the ambiguity by specifying the certificate to use and that you can specify the certificate by changing one of two configuration attributes. 

How does this look to you?  
"Specify which certificate to use in the truststore by adding the alias of the appropriate signer certificate to either the 'trustedAlias' JWT consumer configuration attribute or the 'kevNAME' MP-JWT configuration attribute."

Comment (brutif) [reply dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops]: both changes have been added, thanks.

#build

**Comment by brutif** [comment]
both changes have been added, thanks!

#build

**Comment by brutif** [comment]
build is good.

File: dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops [hunk 3]
```diff
@@ -274,12 +274,12 @@ JWT_SIGNER_CERT_NOT_AVAILABLE=CWWKS6046E: Cannot retrieve a signing key from the
...
 JWT_SIGNER_CERT_AMBIGUOUS=CWWKS6047E: Cannot determine which signer certificate in the truststore to use. Add the 'trustedAlias' attribute to the JWT consumer configuration or the 'keyName' attribute to the MP-JWT configuration.
-JWT_SIGNER_CERT_AMBIGUOUS.explanation=Multiple signer certificates were found in the configured truststore. When there are multiple certificates in the truststore, the 'trustedAlias' attribute must be specified in the JWT consumer configuration or the 'keyName' attribute must be specified in the MP-JWT configuration to establish which of the certificates to use.
-JWT_SIGNER_CERT_AMBIGUOUS.useraction=Specify the 'trustedAlias' JWT consumer configuration attribute or the 'keyName' MP-JWT configuration attribute with the alias of the appropriate signer certificate.
+JWT_SIGNER_CERT_AMBIGUOUS.explanation=Multiple signer certificates were found in the configured truststore. When there are multiple certificates in the truststore, the 'trustedAlias' attribute must be specified in the JWT consumer configuration or the 'keyName' attribute must be specified in the MP-JWT configuration to establish certificate to use.
```
Comment (lzdanski) [inline dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops]: Hi @brutif, it looks like there's a word missing at the end of the .explanation:

"establish **which** certificate to use."

Everything else looks good! Thanks!

Comment (brutif) [reply dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops]: fixed

**Comment by lzdanski** [comment]
Changes look good!

**Comment by brutif** [comment]
ok, done editing, build one last time
#build

**Comment by brutif** [comment]
build is still good.

**Pull request merged**